src/Repository/DatabaseConnectionBuilder.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace BookReviews\Repository;
+
+use \PDO;
+use \PDOException;
+
+/**
+ * Class DatabaseConnectionBuilder
+ * @package BookReviews\Repository
+ */
+class DatabaseConnectionBuilder
+{
+    /**
+     * @return PDO
+     */
+    public function connectToDatabase()
+    {
+        try {
+            $database = new PDO(
+                'mysql:host=127.0.0.1;dbname=BookReviews',
+                'root',
+                ''
+            );
+
+            return $database;
+        } catch (PDOException $e) {
+            echo "Error!: " . $e->getMessage() . "<br/>";
```

Comment:
Try to avoid using echo anywhere outside of the Access Point Interface (controller, index.php) we will discuss alternatives :)